### PR TITLE
RELEASE RPET-269, RPET-567, RPET-559

### PR DESCRIPTION
**Pull request by TSeelig**
[Corresponding div-cos PR](https://github.com/hmcts/div-case-orchestration-service/pull/1320)

- RPET-599 Change Service Method (https://github.com/hmcts/div-ccd-definitions/pull/703)

- RPET-567 update solConfirmPersonalService callback endpoint (https://github.com/hmcts/div-ccd-definitions/pull/722)

- RPET-269 no ccd config changes required

### JIRA link (if applicable) ###
[RPET-269](https://tools.hmcts.net/jira/browse/RPET-269)
[RPET-567](https://tools.hmcts.net/jira/browse/RPET-567)
[RPET-559](https://tools.hmcts.net/jira/browse/RPET-559)

### Change description ###
Combined release PR for RPET-269, RPET-567, and RPET-559

### CODE REVIEW

File: definitions/divorce/json/CaseEvent/CaseEvent.json [hunk 1]
```diff
@@ -773,6 +773,19 @@
...
+  {
+    "LiveFrom": "05/08/2019",
+    "CaseTypeID": "DIVORCE",
+    "ID": "changeServiceMethod",
+    "Name": "Change service method",
+    "Description": "Change the solicitors service method",
+    "DisplayOrder": 1,
+    "PreConditionState(s)": "AosOverdue;AosAwaiting",
+    "PostConditionState": "Issued",
+    "RetriesTimeoutURLAboutToSubmitEvent": "120,120",
```
Comment (yarpo) [inline definitions/divorce/json/CaseEvent/CaseEvent.json:785]: you don't need these fields here.

Please remove
```
"RetriesTimeoutURLAboutToSubmitEvent": "120,120",
"RetriesTimeoutURLSubmittedEvent": "120,120",
```

File: definitions/divorce/json/CaseEvent/CaseEvent.json [hunk 2]
```diff
@@ -2302,7 +2315,7 @@
     "PreConditionState(s)": "AwaitingService",
     "PostConditionState": "AosAwaiting",
     "RetriesTimeoutURLAboutToSubmitEvent": "120,120",
-    "CallBackURLSubmittedEvent": "${CCD_DEF_COS_URL}/bulk-print",
+    "CallBackURLSubmittedEvent": "${CCD_DEF_COS_URL}/solicitor-confirm-service",
```
Comment (yarpo) [inline definitions/divorce/json/CaseEvent/CaseEvent.json:2318]: are you sure this is correct?

File: definitions/divorce/json/CaseEventToFields/CaseEventToFields.json [hunk 3]
```diff
@@ -10294,5 +10294,17 @@
     "PageID": "ReturnToPreviousState1",
     "PageDisplayOrder": 1,
     "PageColumnNumber": 1
+  },
+  {
+    "LiveFrom": "01/01/2017",
```
Comment (yarpo) [inline definitions/divorce/json/CaseEventToFields/CaseEventToFields.json:10299]: I know it's ignored by CCD, but it's easier to investigate when you use current date

File: definitions/divorce/json/CaseEvent/CaseEvent.json [hunk 4]
```diff
@@ -773,6 +773,19 @@
     "RetriesTimeoutURLSubmittedEvent": "120,120",
     "SecurityClassification": "Public"
   },
+  {
+    "LiveFrom": "05/08/2019",
```
Comment (yarpo) [inline definitions/divorce/json/CaseEvent/CaseEvent.json:777]: why this date?

**Review by bendoc1996 — APPROVED** [review]
agree with Patryk's comments, otherwise LGTM!

File: definitions/divorce/json/CaseEvent/CaseEvent.json (hunk 2 above)
Comment (TSeelig) [reply definitions/divorce/json/CaseEvent/CaseEvent.json:2318]: https://github.com/hmcts/div-case-orchestration-service/pull/1320/files#diff-a34f8f77ae27cf30ca8266a79417be90d972cf08b2d6339465816c9e790e7a89R70

File: definitions/divorce/json/CaseEvent/CaseEvent.json (hunk 4 above)
Comment (TSeelig) [reply definitions/divorce/json/CaseEvent/CaseEvent.json:777]: As its ignored by CCD just copied and pasted from event above, will set to current date for clarity

File: definitions/divorce/json/CaseEvent/CaseEvent.json (hunk 1 above)
Comment (TSeelig) [reply definitions/divorce/json/CaseEvent/CaseEvent.json]: Addressed in 4b25a21f5bf186026c7933e86f859a8449132a4f

File: definitions/divorce/json/CaseEventToFields/CaseEventToFields.json (hunk 3 above)
Comment (TSeelig) [reply definitions/divorce/json/CaseEventToFields/CaseEventToFields.json]: Addressed in 4b25a21f5bf186026c7933e86f859a8449132a4f

File: definitions/divorce/json/CaseEvent/CaseEvent.json (hunk 4 above)
Comment (TSeelig) [reply definitions/divorce/json/CaseEvent/CaseEvent.json]: Addressed in 4b25a21f5bf186026c7933e86f859a8449132a4f

**Review by Cotterale — APPROVED** [review]

**Comment by TSeelig** [comment]
failing to build due to corrupted pr pod, green build on: 
https://github.com/hmcts/div-ccd-definitions/pull/828 
https://build.platform.hmcts.net/blue/organizations/jenkins/HMCTS_DIV%2Fdiv-ccd-definitions/detail/PR-826/1/pipeline

**Pull request merged**